README.md

```diff
@@ -46,9 +46,19 @@ We provide a [live demo](https://demo.questdb.io/) provisioned with the latest
 QuestDB release and sample datasets:
 
 - 10 years of NYC taxi trips with 1.6 billion rows
-- live trading data from a cryptocurrency exchange
+- live crytocurrency market data with 30M+ rows per month
 - geolocations of 250k unique ships over time
 
+| Query                                                                         | Execution time                                                                                                                                                                   |
+| ----------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `SELECT sum(double) FROM trips`                                               | [0.15 secs](<https://demo.questdb.io/?query=SELECT%20sum(trip_distance)%20FROM%20trips;&executeQuery=true>)                                                                      |
```

Comment:
The queries are not using the real column names, but generic data type names. Then on the link they use the real query. In some of the queries we are also showing a different query (not just the column names) on the table and on the link. This is very confusing for me

Reply:
Hi @javier please look at https://github.com/questdb/questdb/pull/2780#issue-1450484242
here I explained why the queries are not exactly the same. I tested it and long queries are more confusing.